src/corell/Cargo.toml

```diff
@@ -30,3 +31,7 @@ path = "src/lib.rs"
 bitflags = "0.8"
 log = "0.3"
 draw_state = "0.6"
+
+# TODO: remove
+[dev-dependencies]
+gfx = { path = "../render", version = "0.16" }
```

Comment:
Ok, I removed it. I can't run the vulkan examples obviously, but the behavior seems correct.